src/psyclone/psyad/main.py

```diff
@@ -139,13 +142,30 @@ def msg():
         logger.error("psyad error: file '%s', not found.", filename)
         sys.exit(1)
 
+    # Processing filename
```

Comment:
If you move this check up to L133 we can simplify the associated test as it will no longer need valid filenames for anything.

Reply:
Addressed in b212e079d51685d8d5d5039504774342e92ef6b1.